R/mobr_boxplots.R

```diff
@@ -89,8 +91,16 @@ These are removed for the calculation of PIE."))
    ENS_PIE_sample[plots_n0] = NA
    
    # bias corrected Chao estimator
-   S_asymp_sample = estimateR(mob_in$comm)["S.chao1",]
- 
+   S_asymp_sample = try(vegan::estimateR(mob_in$comm))
+   if (class(S_asymp_sample) == "try_error"){
+      warning("The Chao richness estimator cannot be calculated for all samples.")
+      S_asymp_sample <- rep(NA, nrow(mob_in$comm))
```

Comment:
Are we replacing all sample-level S_Chao with NA's, or only those samples that fail?